bower.json

```diff
@@ -7,6 +7,7 @@
   "dependencies": {
     "chart.js": "https://github.com/nnnick/Chart.js.git#2.0.0-beta2",
     "fontawesome": "4.5.0",
+    "fullcalendar": "2.6.0",
```

Comment:
No Worries :smile: 
The bower reference can be found [here](http://bower.io/).

In this case `bower install fullcalendar --save` will install fullcalendar to the `bower_components` folder, and add it to the `bower.json` file so everyone else will automatically get it installed.

Then remove the full calendar files from the js and css source folders, and point the links in the html to the `bower_components` version.

An example of how that transition would look can be found [here](https://github.com/IMOD-ASU/imod/commit/117dcf4c2920b1a73982de17595a021c67eae564).